doc/classes/Node.xml

```diff
@@ -42,6 +42,7 @@
 			<return type="String">
 			</return>
 			<description>
+				The return value of this method is displayed as a warning in the `Scene Dock` if the script that overrides it is a `tool` script.
```

Comment:
I would also add that returning an empty string produces no warning.

Reply:
Note also that this is wrong formatting for inline code. It should be done with `[code]tool[/code]`. I don't think the "Scene Dock" needs special formatting, it's not a keyword. I would simply write it as "Scene dock".